### DISCUSSION TITLE
Sentry discarding invalid values with a reason of `timestamp out of range`?

### DISCUSSION THREAD

**Discussion by itsramiel** [getsentry/sentry-react-native#3367] [General]
Hi

In some sentry issues I get a red box saying `Sentry has identified the following problems for you to fix` with the following section: `Discarded invalid value (12)`

and then it lists breadcrumbs but without the breadcrumb name and the reason is `timestamp out of range`. Here is a picture:
<img width="1158" alt="image" src="https://github.com/getsentry/sentry-react-native/assets/80689446/e089a47c-5392-45b6-bf5a-55468fe4a602">

Can someone please explain to me what that means? The timestamp is out of what range?

**Comment by jd-carroll** [comment]
Were you ever able to understand this?

**Reply by itsramiel** [reply]
nope

**Reply by jd-carroll** [reply]
https://github.com/getsentry/sentry-javascript/issues/12318

The issue is that you (& myself) are reporting the timestamp in milliseconds, it needs to be in seconds.

https://github.com/getsentry/sentry-javascript/blob/98654721cc6fdb0c222900896c905e6b7bfad289/packages/utils/src/time.ts#L20-L27
